web/owlspa/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "single page application for the owl",
   "scripts": {
     "test:prod": "karma start --single-run",
-    "test:dev": "karma start",
+    "test:dev": "QT_QPA_PLATFORM='' karma start",
```

Comment:
seeing as this is a hack specific to your machine I'd suggest name spacing it as a separate entry.